cms/server/contest/templates/documentation.html

```diff
@@ -8,13 +8,13 @@ <h1>{{ _("Documentation") }}</h1>
 
 <h2>{{ _("Programming languages and libraries") }}</h2>
 
-{% if any(contest.languages|map("get_language")|map(attribute="source_extensions")|select("contains", ".cpp")) %}
+{% if contest.languages|map("get_language")|map(attribute="source_extensions")|select("contains", ".cpp")|list|length > 0 %}
```

Comment:
Can we define our own? like `|nonempty`?

Reply:
I see you defined any later, still nonempty wouldn't be bad for all other occasions you do len() > 0

Reply:
As you mentioned, I added `all` and `any` that make some of these clearer. I think the benefit of `|empty` vs `|length == 0` and `|nonempty` vs `|length > 0` (which, I guess, are brevity and supposed better readability) are not worth the introduction of new filters that may confuse anyone not familiar with our custom idiom.